ocs_ci/ocs/constants.py

```diff
@@ -672,6 +672,7 @@
 
 # Deployment constants
 OCS_CSV_PREFIX = "ocs-operator"
+LOCAL_STORAGE_SUBSCRIPTION = "local-storage-operator"
```

Comment:
Do we need this? We already have a constant for `local-storage-operator` on the next line. If the constant name is the issue we could always make this more generic. `LOCAL_STORAGE_OPERATOR` or something like that.

Reply:
Oh, if `LOCAL_STORAGE_CSV_PREFIX` has the same effective meaning as `LOCAL_STORAGE_SUBSCRIPTION` I'm fine with that.